sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConfig.scala

```diff
@@ -872,7 +874,8 @@ private object CosmosWriteConfig {
       "ignore pre-existing items i.e., Conflicts), `ItemDelete` (deletes based on id/pk of data frame), " +
       "`ItemDeleteIfNotModified` (deletes based on id/pk of data frame if etag hasn't changed since collecting " +
       "id/pk), `ItemOverwriteIfNotModified` (using create if etag is empty, update/replace with etag pre-condition " +
-      "otherwise, if document was updated the pre-condition failure is ignored)")
+      "otherwise, if document was updated the pre-condition failure is ignored)," +
+      " `ItemPatchBulkUpdate` (read item, then patch the item locally, then using create if etag is empty, update/replace with etag pre-condition)")
```

Comment:
I think it would be good to elaborate a bit on automatic retries - I would assume if a competing update between read many and create or replace happens we would automatically retry and start over from read many - so, this is informative for customers to understand how we achieve that we can safely patch the document. But nothing they should need to worry about.

Reply:
updated